br compact backup user document (#2676)
pingcap/docs-tidb-operator: en/backup-restore-cr.md

* `.spec.local.volume`: the persistent volume configuration.
* `.spec.local.volumeMount`: the persistent volume mount configuration.

> ## CompactBackup CR fields

Reviewer: ```suggestion
## CompactBackup CRD fields
```

* `.spec.local.volume`: the persistent volume configuration.
* `.spec.local.volumeMount`: the persistent volume mount configuration.

## CompactBackup CR fields

> To compact log backup data into structured SST files, users can create a custom CompactBackup CR object to describe a backup task. Below is a detailed explanation of the fields in the CompactBackup CR:

Reviewer: add a sentence saying "From TiDB version 9.0.0 and above, you can compact log backup to accelerate PITR restore。"

> * `.spec.startTs`: Specifies the starting timestamp for the log compression backup range.

Reviewer: ```suggestion
* `.spec.startTs`: range Defines the time range for the compaction operation.
```

Reviewer: please sync with the terms https://github.com/pingcap/docs/pull/20342/files. ditto for other occurrence.

Author: ok

> For ***TiDB version 9.0.0*** and above, you can compact log backup to accelerate PITR restore. To compact log backup data into structured SST files, users can create a custom CompactBackup CR object to describe a backup task. Below is a detailed explanation of the fields in the CompactBackup CR:

Author: ```suggestion
For TiDB version 9.0.0 and above, you can compact log backup to accelerate PITR restore. To compact log backup data into structured SST files, users can create a custom CompactBackup CR object to describe a backup task. Below is a detailed explanation of the fields in the CompactBackup CR:
```

* `.spec.endTs`: Specifies the ending timestamp for the log compression backup range.
* `.spec.concurrency`: Maximum number of log compression tasks that can run simultaneously. Defaults to `4`
* `.spec.maxRetryTimes`: Maximum number of retries for failed compression tasks. Defaults to `6`

> * `.spec.toolImage`：the tools image used by `Restore`. The only tool image used in `CompactBackup` is BR.

Author: do ne

    - If not specified or left empty, the `pingcap/br:${tikv_version}` image is used for backup by default.

>     - If a BR version is specified, such as `.spec.toolImage: pingcap/br:v8.1.0`, the image of the specified version is used for backup.

Reviewer 2: ```suggestion
    - If a BR version is specified, such as `.spec.toolImage: pingcap/br:v8.5.0`, the image of the specified version is used for backup.
```

>     - If an image is specified without a version, such as `.spec.toolImage: private/registry/br`, the `private/registry/br:${tikv_version}` image is used for backup.

Reviewer 2: ```suggestion
    - If an image is specified without a version, such as `.spec.toolImage: private/registry/br`, the `private/registry/br:${tikv_version}` image is used for backup.

```

* `backupTemplate`: the configuration of the snapshot backup. Specifies the configuration related to the cluster and remote storage of the snapshot backup, which is the same as the `spec` configuration of [the `Backup` CR](#backup-cr-fields).
* `logBackupTemplate`: the configuration of the log backup. Specifies the configuration related to the cluster and remote storage of the log backup, which is the same as the `spec` configuration of [the `Backup` CR](#backup-cr-fields). The log backup is created and deleted along with `backupSchedule` and recycled according to `.spec.maxReservedTime`. The log backup name is saved in `status.logBackup`.

> * `compactBackupTemplate`: the configuration template of the log compaction backup. The fields are the same as those in the `spec` configuration of [the `CompactBackup` CR](#compactbackup-cr-fields). The compaction backup is created and deleted along with `backupSchedule`. The storage settings of the compaction backup should be the same as that of `logBackupTemplate` in the same `backupSchedule`.

Reviewer 2: ```suggestion
* `compactBackupTemplate`: the configuration template of the log compaction backup. The fields are the same as those in the `spec` configuration of [the `CompactBackup` CR](#compactbackup-cr-fields). The compaction backup is created and deleted along with `backupSchedule`. The log backup names are stored in `status.logBackup`. The storage settings of the compaction backup should be the same as that of `logBackupTemplate` in the same `backupSchedule`.
```

    * `.spec.maxBackups`: a backup retention policy, which determines the maximum number of backup files to be retained. When the number of backup files exceeds this value, the outdated backup file will be deleted. If you set this field to `0`, all backup items are retained.
    * `.spec.maxReservedTime`: a backup retention policy based on time. For example, if you set the value of this field to `24h`, only backup files within the recent 24 hours are retained. All backup files older than this value are deleted. For the time format, refer to [`func ParseDuration`](https://golang.org/pkg/time/#ParseDuration). If you have set `.spec.maxBackups` and `.spec.maxReservedTime` at the same time, the latter takes effect.
    * `.spec.schedule`: the time scheduling format of Cron. Refer to [Cron](https://en.wikipedia.org/wiki/Cron) for details.

>     * `.spec.compactInterval`: The interval duration that determines when a new compaction should be triggered. This field works in two scenarios:

Author: This rule need update

Author: done
